packages/main/src/plugin/tasks/progress-impl.ts

```diff
@@ -78,14 +83,34 @@ export class ProgressImpl {
     );
   }
 
+  protected getTaskAction(options: extensionApi.ProgressOptions): TaskAction | undefined {
+    if (!options.navigation) return undefined;
+
+    if (!this.navigationManager.hasRoute(options.navigation.routeId)) {
+      console.warn(`cannot created task action for unknown routeId ${options.navigation.routeId}`);
+      return undefined;
+    }
+
+    return {
+      name: 'Goto task >',
```

Comment:
```suggestion
      name: 'Go to task >',
```

Reply:
I would just use `Details` or `More details`

Reply:
Stevan added a comment, and have a preference for `view` it is okay for you ?